docs/deployment/universal-profile.md

```diff
@@ -57,7 +57,7 @@ await lspFactory.UniversalProfile.deploy({
 ```
 
 :::info Info
-Profile Metadata can be passed as either a JSON object containing the [LSP3Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-3-UniversalProfile-Metadata.md#lsp3profile) you want to upload or an IPFS URL of your previously uploaded metadata.
+Profile Metadata can be passed as either a JSON object containing the [LSP3Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-3-UniversalProfile-Metadata.md#lsp3profile) you want to upload or a URL of your previously uploaded metadata.
```

Comment:
Maybe two **bullet points** would make it easier to see the 2 available options?

```markdown
Profile Metadata can be passed in two ways:
- as a JSON object containing the [LSP3Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-3-UniversalProfile-Metadata.md#lsp3profile) you want to upload.
- as a URL of your previously uploaded metadata.
```